app/assets/javascripts/discourse/lib/utilities.js.es6

```diff
@@ -410,5 +410,46 @@ export function rescueThemeError(name, error, api) {
   document.body.prepend(alertDiv);
 }
 
+const CODE_BLOCKS_RULES = [
+  { rule: "^(    |\\t).*", end: "" },
+  { rule: "`.+?`", end: "`" },
```

Comment:
I wonder if this one is any faster?

```suggestion
  { rule: "`[^`]+`", end: "`" },
```

Reply:
They are both super-fast, but your version looks more readable.